libs/angular-components/tedi/services/translation/translation.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable, signal } from "@angular/core";
+import { translationsMap, TranslationMap, TediTranslationsMap } from "./translations";
+
+export type Language = "en" | "et" | "ru";
+
+@Injectable({ providedIn: "root" })
+export class TranslationService {
```

Comment:
maybe should use some kind of prefix here in naming? i would assume that a lot of applications already use ngx-translate that has same naming